services/src/test/java/edu/unc/lib/dl/cdr/services/rest/RunEnhancementsIT.java

```diff
@@ -108,10 +127,12 @@ public void setup() throws Exception {
 
     @Test
     public void runEnhancementsFileObject() throws Exception {
+        FileObject fileObj = repositoryObjectFactory.createFileObject(PIDs.get(UUID_TEST), null);
```

Comment:
If you call `createFileObject(null)` then it will make a PID for you and you don't have to worry about collisions between test runs. You can get the pid back by calling `fileObj.getPid()`. This would allow you to remove hard coded pids too.